### PR TITLE
Microverse Trails Code: Review

### PR DESCRIPTION
**Pull request by porag-m06**
Files from my microverse trail are added to perform code review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Project-From-Trials-
\ No newline at end of file
+# Microverse-Trial---Salman-Porag
```
Comment (porag-m06) [inline README.md]: Please follow the descriptive README template suggested by Microverse. As the README file is the face of your project therefore it is very important to make it very descriptive with all the details of your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js"></script>
+    <title>Document</title>
```
Comment (porag-m06) [inline index.html:11]: Please make the title more meaning full (e. g. Home) to properly indicate the functionality of the page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    </section>
+
+    <section id="pricing" class="section-class">
+        <h1>Handcrafted, home-made masterpieces</h1>
+
+<form id="form" action="https://www.freecodecamp.com/email-submit">
+    <input id="email" name="email" type="email" placeholder="enter your email here" required >
+    <input id="submit" class="btn btn-primary" type="submit"></input>
+</form>
+    </section>
```
Comment (porag-m06) [inline index.html:78]: Please have all the section tags under the semantic tag [ <main> </main>] as it helps the search engines and assistive technologies (e.g. screen readers) to determine the importance and context of web pages

File: index.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    <section id="howitwork" class="section-class">
+        <iframe id="video" width="420" height="315" src="https://www.youtube.com/embed/tgbNymZ7vqY" frameborder="0">
+        </iframe>
+    
+    </section>
+
+    <section id="pricing" class="section-class">
+        <h1>Handcrafted, home-made masterpieces</h1>
+
+<form id="form" action="https://www.freecodecamp.com/email-submit">
```
Comment (porag-m06) [inline index.html:74]: Please use the form's action attribute according to your project requirements. It should not submit the from on a generic link that has no meaningful connection to your project.

**Review by porag-m06 — COMMENTED** [review]
Hi @porag-m06,

Good job so far!
There are some issues, however, that you still need to be addressed to go to the next project but you are almost there! :) 

image

# Highlights
- Correct GitHub Flow ✔️
- Good implementations ✔️

# Required Changes ♻️
 [REQUIRED] Good job so far!  However, the linter setup and check on GitHub action is not done which needs to be added and should be passed all the checks to green. AKA: Linter should report NO errors, please fix your linter errors 🙏🏼

Check the comments under the review.

# Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Olamarx — APPROVED** [review]
Hi @porag-m06, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- GitHub review features ✔️
- Inline review ✔️ 
- Polite comments ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**